### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by louieQ**
I wrote the documentation where missing.

Please Giovanni, I need your help to complete the docstring for a couple of methods.

### CODE REVIEW

File: KGTorrent/populate_db.py [hunk 1]
```diff
@@ -98,12 +121,17 @@ def __init__(self, constraints_file_path, meta_kaggle_path, sqlalchemy_engine):
...
+        ``ForumMessageVotes`` and ``Submissions``.
+
 
         Args:
-            table_name:
+            table_name: the name of the ``.csv`` table to be loaded from the Meta Kaggle dataset.
             is_ready_for_db:
-
-        Returns:
+            TODO: write documentation for the is_ready_for_db argument.
```
Comment (louieQ) [inline KGTorrent/populate_db.py:134]: Here I couldn't write the documentation for the `is_ready_for_db` argument.

File: KGTorrent/populate_db.py [hunk 2]
```diff
@@ -163,12 +191,12 @@ def load_table(self, table_name, is_ready_for_db=False):
...
         """
+        This method writes preprocessed tables to the database by leveraging the ``pandas.DataFrame.to_sql`` method.
 
         Args:
-            table_name:
+            table_name: the name of the table to be written to the database.
             df:
-
-        Returns:
+            TODO: write documentation for the df argument.
```
Comment (louieQ) [inline KGTorrent/populate_db.py:199]: Here I couldn't write the documentation for the `df` argument.

**Comment by louieQ** [comment]
Moreover, in `KGTorrent.downloader` the SQL statement is not complete. Does it work anyway? How?

(the condition of the `WHERE` statement is empty).

**Comment by Giovalo** [comment]
As we discussed in the meeting, everything is fine for me.

**Pull request merged**